IM_calculation/IM/read_waveform.py

```diff
@@ -240,7 +237,7 @@ def read_binary_file(
     """
     waveforms = []
     # convert str comps to integer
-    comps = [COMP_DICT[c] for c in comps]
+    # comps = [COMP_DICT[c] for c in comps]
```

Comment:
Is there plans to use this in the future? Otherwise remove it?

Reply:
This was part of a temporary patch while partially refactored towards using the enum